pipeline/run_benchmark.sh

```diff
@@ -200,9 +210,29 @@ EOT
       echo "$TIMEOUT -s SIGINT ${CURL_TIMEOUT} $CMD > $LOGDIR/$cmd/$cmd.$n.$ip.log" >> $cmd_file
    done
 
-   start_index=${configs[benchmark.shards]}
+# send commands to explorers at second
+   local explorer_shard_id=0
+   local explorer_start_index=${configs[benchmark.shards]} + 1
```

Comment:
this won't work, use $(( ${configs[benchmark.shards]} + 1 ))

Reply:
Fixed.